src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -1001,6 +1006,34 @@ def __repr__(self):
         return repr_str
 
 
+class ConvertBox(tvt_v2.Transform):
```

Comment:
What do you think about using this functionality in head (decoder) when calculating loss like the below instead of this new transform module?
I think we can execute the same operation in target device (GPU for training) and reduce time even if it takes very little time.
https://github.com/openvinotoolkit/training_extensions/blob/debd5e05d9175bf47705abc6e24f8cdb61097bf9/src/otx/algo/detection/heads/yolox_head.py#L674

Reply:
I added it in the convert_inputs stage. bbox scaling is also required here. I deleted ConvertBox 
https://github.com/openvinotoolkit/training_extensions/pull/3741/commits/d5b77c306e41641e9616d5d3975f6b84d5e3f390